share/extender/build_input.yml

```diff
@@ -236,7 +236,7 @@ platforms:
 
   arm64-ios:
     env:
-        SYSROOT:    "{{env.PLATFORMSDK_DIR}}/iPhoneOS{{env.IOS_17_VERSION}}.sdk"
+        SYSROOT:    "{{env.PLATFORMSDK_DIR}}/iPhoneOS{{env.IOS_18_VERSION}}.sdk"
```

Comment:
Maybe here and below we can use `{{ env.IOS_VERSION }}`? Helps avoid changing every time that place

Reply:
this is `env` and we specify it in `env` section, do we really apply `env` templates to itself?

Reply:
Good question. Let me check locally.